packages/@webex/internal-plugin-device/src/device.js

```diff
@@ -696,6 +696,8 @@ const Device = WebexPlugin.extend({
       this.refreshTimer = safeSetTimeout(() => this.refresh(), delay);
     }
 
+    this.webex.internal.newMetrics.callDiagnosticMetrics.setDeviceInfo(this);
```

Comment:
I've done some testing and it seems that CA events pre-login, are missing machineId and userId form the identifiers object.
### localhost
<img width="1668" alt="image" src="https://github.com/webex/webex-js-sdk/assets/14194394/e100e3f3-1354-4441-8e2e-89f6fa0bc86a">

### web.webex.com
<img width="1674" alt="image" src="https://github.com/webex/webex-js-sdk/assets/14194394/5a05e4f8-0b35-4a28-b4d0-5d7a8adc057f">

Reply:
Not sure why, but seems related to the change in the PR.

Reply:
Aslo, this line change is missing tests.

Reply:
@torgeadelin - I can't seem to reproduce this one. See screenshots:

This is when I entered my PMR and logged in: 
![Screenshot 2024-06-10 at 6 03 06 AM](https://github.com/webex/webex-js-sdk/assets/8044050/083b23fd-fef9-4568-a65a-311acaf3306f)

This is when I just loaded `localhost:4000/meeting` that redirected automatically to `localhost:4000/join-a-meeting`:
![Screenshot 2024-06-10 at 6 04 34 AM](https://github.com/webex/webex-js-sdk/assets/8044050/47e8dbff-f77b-4cdf-a925-6a21d7534585)

Reply:
Locally, did you link both the plugins?
```
"@webex/internal-plugin-device": "link:/Users/kmadavan/Documents/repos/web-sdk/circularDepV2/packages/@webex/internal-plugin-device",
"@webex/internal-plugin-metrics": "link:/Users/kmadavan/Documents/repos/web-sdk/circularDepV2/packages/@webex/internal-plugin-metrics",
```